### PR TITLE
ipe: Bump version and fix qhull linking

### PR DESCRIPTION
**Pull request by rubendibattista**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.x
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by rubendibattista** [comment]
Mmmh, I don't understand the problem with 8.3. Maybe `stdlib.h` must be included for older versions?

**Comment by rubendibattista** [comment]
@kencu

File: graphics/ipe/Portfile [hunk 1]
```diff
@@ -22,9 +22,10 @@ master_sites            https://dl.bintray.com/otfried/generic/ipe/${branch}
 distname                ${name}-${version}-src
 worksrcdir              ${name}-${version}/src
 
-checksums           sha1    58410e29c04e7a7c3978bd9afe6d25a7924bd30d \
-                    rmd160  6d68ad7831923d04ba07086f3dde2be9830d756a \
-                    sha256  d66d3f2619e3e6ff617f42c2e3695c3db6e2a64adcc3d7613214e5fd14c49f22
+checksums           sha1    c8b82b6dc8bc286ad8f610f01e954e2bdd6ac65d \
```
Comment (cjones051073) [inline graphics/ipe/Portfile]: Please remove the sha1 checksum. These days, we favour rmd160, sha256 and size only.

**Comment by mf2k** [comment]
The commit message should explicitly mention the version number.

**Comment by rubendibattista** [comment]
Something else missing?

**Comment by cjones051073** [comment]
Looks OK I suppose, although you should report back upstream the build failure

```
ipeluaipelet.cpp:173:38: error: use of undeclared identifier 'getenv'
    ipeDebug("DYLD_LIBRARY_PATH=%s", getenv("DYLD_LIBRARY_PATH"));
```

looks likely just a missing include. for <stdlib.h>

**Comment by rubendibattista** [comment]
@cjones051073 I believe that problem is related to some implementation differences of `stdlib` in old OSX systems. I'm not sure this will be taken care upstream. I'll try to ask, if not I'll provide a patch (but I cannot test it so I'll rely on buildbot logs)

**Comment by cjones051073** [comment]
Xcode 8.3 is not that old, and I am pretty sure all macOS systems have `getenv`. So I am not convinced this is what you say....

**Comment by cjones051073** [comment]
Issue was indeed what I said, a missing include (it must be implicitly included by the compiler in the cases where it currently builds OK). Fixed in https://github.com/macports/macports-ports/commit/5a492077ab90915e28fdf7f0bf7e44cc87099e7d

**Comment by rubendibattista** [comment]
@cjones051073 With "implementation differences" in my head I was referring to implicit including `stdlib` in newer toolchain version. 

Thank you in any case for your fix! :)

**Pull request merged**